packaging/amazon-linux-ami/ecs.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=ECS Agent
+Requires=docker.service
+After=docker.service
+
+[Service]
```

Comment:
I wouldn't mind seeing `Type=simple` set explicitly here. Otherwise people will need to understand the rules around the different defaults in order to understand this unit's behavior.